Utils/Pieces/Pawn.py

```diff
@@ -7,35 +7,35 @@ def __init__(self, position,color):
         self.photo="Images/wp.png" if color == "white" else "Images/bp.png"
 
     def can_move(self,board):
-        # Ten ruch poprawie kiedy bede dodawał roszade
-        output = []
+        if self.color=="white":
+            poss_change=-1
```

Comment:
`poss_change`? Would change name of this variable. What is it? `position_change`?